src/org/labkey/targetedms/TargetedMSController.java

```diff
@@ -4836,14 +4840,17 @@ public static Integer addProteinSummaryViews(VBox box, PeptideGroup group, Targe
         if (group.getSequenceId() != null)
         {
             int seqId = group.getSequenceId().intValue();
-            List<String> peptideSequences = new ArrayList<>();
+            List<PeptideCharacteristic> peptideCharacteristics = new ArrayList<>();
             for (Peptide peptide : PeptideManager.getPeptidesForGroup(group.getId()))
             {
-                peptideSequences.add(peptide.getSequence());
+                var peptideCharacteristic = PeptideManager.getPeptideCharacteristic(peptide.getId());
+                peptideCharacteristic.setIntensityView(group.getViewBy() != null && group.getViewBy().equalsIgnoreCase("intensity"));
```

Comment:
Similar to my comment in the related PR, this doesn't feel like it should be a peptideCharacteristic level setting. They're all going to have the same value